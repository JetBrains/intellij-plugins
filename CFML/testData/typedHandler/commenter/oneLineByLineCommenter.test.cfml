<cfscript>
	var a = <caret>2.4;
</cfscript>