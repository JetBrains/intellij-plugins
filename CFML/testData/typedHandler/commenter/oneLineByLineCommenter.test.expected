<cfscript>
	//var a = 2.4;
</cfscript>